test/server/util/twilio_mocks.js

```diff
@@ -22,25 +25,70 @@ export function callStatusUpdate() {
     priceUnit: 'USD',
     sid: 'CAcbbf06f666c72c51c59200de56ae54ff',
     startTime: '2017-09-11T18:20:30.000Z',
-    status: 'completed',
+    status: null,
     subresourceUris: {
       notifications: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff/Notifications.json`,
       recordings: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff/Recordings.json`
     },
     to: '+14086428264',
     toFormatted: '(408) 642-8264',
     uri: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/CAcbbf06f666c72c51c59200de56ae54ff.json`
-  };
+  }
+  switch (updateType) {
+    case CALL_COMPLETE:
+      clientUpdateObject.duration = '14';
+      clientUpdateObject.endTime = '2017-09-11T18:20:44.000Z'
+      clientUpdateObject.status = 'completed'
+      return clientUpdateObject;
+    case CONNECT_CONTACT_URL:
+      clientUpdateObject.status = 'in-progress'
+      return clientUpdateObject;
+    default:
+      return clientUpdateObject;
+  }
 }
 
 export function createCall() {
   // Allie's create call callback Object goes here
 }
 
+//stole this array comparison extension from stackoverflow
+Array.prototype.equals = function (array) {
```

Comment:
If this is only being used in this file it might be better to just create a helper function rather than adding to the Array prototype.
I may be reading this wrong, but there doesn't seem to be a way to check equivalence of an array of objects, which we have a lot of in this project.

Reply:
Also we have lodash in our dependencies and you can just use _.isEqual.